File: common/src/main/java/com/fuusy/common/utils/ToastUtil.java
package com.fuusy.common.utils;

import android.text.TextUtils;
import android.widget.Toast;

public class ToastUtil {
    private static Toast toast;

    public static void show(String msg) {
        if (TextUtils.isEmpty(msg) || AppUtil.INSTANCE.getApplication() == null) {
            return;
        }
        if (toast == null) {
            toast = Toast.makeText(AppUtil.INSTANCE.getApplication(), "", Toast.LENGTH_SHORT);
        }
        toast.setText(msg);
        toast.show();
    }
}

File: common/src/main/java/com/fuusy/common/utils/ToastUtil.kt
package com.fuusy.common.utils

import android.text.TextUtils
import android.widget.Toast
import com.fuusy.common.utils.AppUtil.getApplication

object ToastUtil {
    private var toast: Toast? = null
    fun show(msg: String?) {
        if (TextUtils.isEmpty(msg) || getApplication() == null) {
            return
        }
        if (toast == null) {
            toast = Toast.makeText(getApplication(), "", Toast.LENGTH_SHORT)
        }
        toast!!.setText(msg)
        toast!!.show()
    }
}